client-side/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import './App.scss';
+import DateTimePicker from './components/report.jsx'
+import VisitedWebsitesComponent from './graphs/graphs.jsx';
+function App() {
+
+  const [startDate, setStartDate] = useState('');
```

Comment:
you shouldn't add code to the app, please move it to statistics component.
and also why do you have this App.js file, we have already App.jsx